service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentReferencesComponentTest.java

```diff
@@ -124,7 +125,7 @@ private GetGpcDocumentReferencesTaskDefinition buildFindDocumentTask(EhrExtractS
     }
 
     private GetGpcDocumentTaskDefinition buildGetDocumentTask(GetGpcDocumentReferencesTaskDefinition taskDefinition) {
-        var documentUrl = buildDocumentUrl();
+        var documentUrl = buildDocumentUrl(taskDefinition.getFromOdsCode());
```

Comment:
I think this needs to be toOdsCode - fromOdsCode being the GP who we received the EhrRequest "from" and toOdsCode being who the request is addressed to. From is winning practice, to is loosing practice

Reply:
@mattd-kainos the ticket says:
_**THEN** the ODS code defined by the element in url’s path 
http:/{gps-consumer-url}/{ods-code}/STU3/1/gpconnect
should the same as provided in EhrExtractRequest value of the medical practice the patient is moved from. (EhrExtractStatus.EhrRequest.fromOdsCode)_

just double checking if the description is wrong and should be corrected?

Reply:
The naming of these variables is tricky and could be improved. 

"the medical practice the patient is moved from" is the target of the EhrRequest to from the perspective of this message, the toOdsCode